pkg/oc/cli/cmd/importimage.go

```diff
@@ -87,24 +64,52 @@ type ImportImageOptions struct {
 	CommandName string
 
 	// helpers
-	out         io.Writer
-	errout      io.Writer
 	imageClient imageclient.ImageInterface
 	isClient    imageclient.ImageStreamInterface
 }
 
+func NewImportImageOptions(name string, streams genericclioptions.IOStreams) *ImportImageOptions {
+	return &ImportImageOptions{
+		IOStreams:       streams,
+		CommandName:     name,
```

Comment:
do I even want to know why they need this?

Reply:
Ah, was not being used - removed